compiler/rustc_ast_passes/src/feature_gate.rs

```diff
@@ -77,13 +76,13 @@ impl<'a> PostExpansionVisitor<'a> {
     fn check_abi(&self, abi: ast::StrLit) {
         let ast::StrLit { symbol_unescaped, span, .. } = abi;
 
-        match abi::is_enabled(self.features, span, symbol_unescaped.as_str()) {
+        match rustc_abi::is_enabled(self.features, span, symbol_unescaped.as_str()) {
             Ok(()) => (),
-            Err(abi::AbiDisabled::Unstable { feature, explain }) => {
+            Err(rustc_abi::AbiDisabled::Unstable { feature, explain }) => {
                 feature_err_issue(&self.sess, feature, span, GateIssue::Language, explain).emit();
             }
-            Err(abi::AbiDisabled::Unrecognized) => {
-                if self.sess.opts.pretty.is_none_or(|ppm| ppm.needs_hir()) {
+            Err(rustc_abi::AbiDisabled::Unrecognized) => {
+                if self.sess.opts.pretty.map_or(true, |ppm| ppm.needs_hir()) {
```

Comment:
what happened here?

Reply:
pls change this back to `is_none_or` and amend whatever commit this change comes from, then r=me

Reply:
...I have no idea, tbh!

Reply:
my best guess is that git decided this was correct after a rebase